server/testutils/factories/cas1NonArrival.ts

```diff
@@ -0,0 +1,10 @@
+import { Factory } from 'fishery'
+import { Cas1NonArrival } from '@approved-premises/api'
+import { faker } from '@faker-js/faker'
+import { DateFormats } from '../../utils/dateUtils'
+
+export default Factory.define<Cas1NonArrival>(() => ({
+  reason: faker.string.uuid(),
+  notes: faker.lorem.words(20),
+  confirmedAt: DateFormats.dateObjToIsoDateTime(new Date(faker.date.recent())),
```

Comment:
Faker does return a `Date` object so the `new Date()` is unnecessary here:

```suggestion
  confirmedAt: DateFormats.dateObjToIsoDateTime(faker.date.recent()),
```

Reply:
True